src/main/java/com/futurcollars/lesson11/zad7/BasicSalary.java

```diff
@@ -0,0 +1,6 @@
+package com.futurcollars.lesson11.zad7;
+
+public interface BasicSalary {
+
+     int getBasicSalary(int basicSalary );
```

Comment:
argument niepotrzebny, co to za serwis, któremu musisz podać liczbę, żeby Ci ją zwrócić? bardziej pasuje mi podanie na przykład jakiegoś id pracownika, ale to nie tutaj :p